radis/lbl/broadening.py

```diff
@@ -888,11 +893,20 @@ def _calc_broadening_HWHM(self):
         # diluent and their broadening coeff dictionary
         diluent_broadening_coeff = {}
         for key in diluent:
-            if key != "air":
+            try:
                 diluent_broadening_coeff["gamma_" + key.lower()] = df[
                     "gamma_" + key.lower()
                 ]
                 diluent_broadening_coeff["n_" + key.lower()] = df["n_" + key.lower()]
+            except:
+                    import warnings
+                    from radis.misc.warning import AccuracyWarning
+                    if key!= "air":
+                        warnings.warn(
+                            AccuracyWarning(
+                                "Broadening Coefficient of "+key+" not present in database using broadening coeffiecient of air instead."
```

Comment:
within SpectrumFactory I think you can use `self.warn` ; have a look at how it's done elsewhere in the code

Reply:
done